meetup/services/meetup_presenter.py

```diff
@@ -0,0 +1,42 @@
+import time
+from pizzaplace.services.pizza_place_presenter import PizzaPlacePresenter
+from pizzaplace.services.yelp_api import YelpApi
+
+class MeetupPresenter():
+  def __init__(self, meetup, meetup_api, api_response_parser):
+    self.meetup = meetup
+    self.parsed_api_response = self.get_api_response(meetup_api, api_response_parser)
+
+  def get_meetup_link(self):
+    return self.meetup.meetup_link
+
+  def get_meetup_name(self):
+    return self.meetup.name
+
+  def get_meetup_venue(self):
+    return self.parsed_api_response.get('venue')
+
+  def get_meetup_next_event_topic(self):
+    return self.parsed_api_response.get('next_event_topic')
+
+  def get_meetup_datetime(self):
+    time_string = self.parsed_api_response.get('datetime')
+    return time.ctime(int(time_string))[:-6]
```

Comment:
I have no idea what `time.ctime` does but I would if this cryptic line was a helper method with a name. I guess it's probably something like `convert_unix_timestamp_to_human_readable_time`, but who knows!